packages/hdwallet-ledger/package.json

```diff
@@ -31,9 +31,9 @@
     "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@ledgerhq/hw-app-btc": "^10.4.1",
-    "@ledgerhq/hw-app-eth": "^6.38.0",
-    "@ledgerhq/hw-transport": "^6.31.2",
+    "@ledgerhq/hw-app-btc": "^10.10.0",
```

Comment:
just noticing, we are actually importing from this package, not just types

Reply:
https://github.com/shapeshift/hdwallet/pull/726/commits/fc2284cf25d937836fc3d2c61de8c4018e15b160